components/expenses/ExpenseForm.js

```diff
@@ -349,6 +357,18 @@ const ExpenseFormBody = ({
     }
   }, [values.payeeLocation]);
 
+  React.useEffect(() => {
+    if (values.payoutMethod?.type === PayoutMethodType.PAYPAL) {
+      if (!PayPalSupportedCurrencies.includes(values.currency)) {
+        formik.setFieldValue('currency', 'USD');
```

Comment:
Why default to USD?

Reply:
@Betree : This was added previously so that when we change the payout method to PayPal but then the default currency of the collective isn't in the PayPal supported currencies (say for example LKR) we show `USD`. With the current implementation when this happens the first currency in the list of paypal supported currencies (AUD) is shown. I don't think this is a problem, but in my implementation I thought that it would be nice to show USD as it's normally the currency that people pay with compared to AUD. 

[paypal-test.webm](https://user-images.githubusercontent.com/12435965/214909679-e8ec45df-f9ec-435b-bfbe-5e69cbeb276c.webm)

Reply:
Makes sense, it's a bit edge case but I'll add back support for this